### PR TITLE
Fix QB Orders in Chinese Lang

### PR DESCRIPTION
**Pull request by KoolShow**
After some commits the order of those task has been changed, and the translation was not changed to be satisfied. I was annoyed when playing nomi-ceu so I fix them in normal mode. For example, this is `nomifactory.quest.normal.db.835.title`
### Before

Whose title unexceptionally shows `MV Blender`.

### After

Now it should be `Oil Cracking Unit`

I didn't edit the Expert mode questbook because I'm not an expert.

### CODE REVIEW

**Comment by KoolShow** [comment]
I'm sorry for making so many changes, since it's all order number changes. Apparently diff is not good at showing this kind of change.

File: nomi-ceu/assets/questbook/lang/zh_cn.lang [hunk 1]
```diff
@@ -2472,1057 +2472,1057 @@ nomifactory.quest.normal.db.780.desc=一旦你获得了你的第一个§6下界
 nomifactory.quest.normal.db.781.title=通货能量
 nomifactory.quest.normal.db.781.desc=§6通货能源炉§r是整合包游戏中期的主要能量来源。用一个§a宝石印刻校准§r和一个自给自足的§b深度怪物学习§r搭配安装，这个能源炉可以烧掉§6钻石§r来产生大量的§eRF能量§r。%n%n就像其他§b热力膨胀§r能源炉一样，§6通货能源炉§r可以被赋予升级套件，以产生更多的能量，并获得更多的§a增强槽位§r。
 
-# db 782 of mode normal
```
Comment (IntegerLimit) [inline nomi-ceu/assets/questbook/lang/zh_cn.lang:2475]: What happened to the translations for quest id 782?

Comment (zero6six) [reply nomi-ceu/assets/questbook/lang/zh_cn.lang:2475]: Large Turbines -> High Amperage Energy Hatches
There was no translation for High Amperage Energy Hatches before. I'll get this one up after I finish looking at the other quests.

File: nomi-ceu/assets/questbook/lang/zh_cn.lang [hunk 2]
```diff
@@ -2472,1057 +2472,1057 @@ nomifactory.quest.normal.db.780.desc=一旦你获得了你的第一个§6下界
...
+nomifactory.quest.normal.db.1042.desc=§3有源变压器§r是一种可以将任意数量和类型的§dGT能量输入§r转换为任意数量和类型的§dGT能量输出§r的装置。这包括§664A蓄能变电站舱口§r，以及§a激光仓§r——参考附近的任务了解更多细节。%n%n虽然因为这个整合包里有§dRF导管§r和§3能量转换器§r，它可能用途有限，但你可以建造它，如果你想要的话。
 
 # db 1043 of mode normal
-nomifactory.quest.normal.db.1043.title=§2激光网络
-nomifactory.quest.normal.db.1043.desc=§a激光§r是§d格雷科技§r针对大功率能量传输的原生解决方案。从§1IV§r开始的每个电压层都有§a激光仓§r，能够传输§d256A§r，§d1024A§r和§d4096A§r的电压功率！%n%n激光网络中的能量从§a激光源仓§r通过§6激光传导线缆§r流向§a激光靶仓§r。所有这些都必须在一条直线上——毕竟激光不会弯曲。与其他与功率相关的块类似，§a激光靶仓§r的电压等级不得低于§a激光源仓§r。%n%n当在§3有源变压器§r和§3蓄能变电站§r之间移动电力时，您可以选择使用这些。
+nomifactory.quest.normal.db.1043.title=太阳能板VIII
+nomifactory.quest.normal.db.1043.desc=八级§6太阳能板§r，这是制作更高层的太阳能板所必需的。这也是一个不错的电源。
 
 # db 1044 of mode normal
-nomifactory.quest.normal.db.1044.title=存储板条箱
```
Comment (zero6six) [inline nomi-ceu/assets/questbook/lang/zh_cn.lang:3524]: this should be replaced with 1045

**Pull request merged**